src/main/java/pl/sda/library/domain/model/exception/AuthorDoesNotExistException.java

```diff
@@ -0,0 +1,7 @@
+package pl.sda.library.domain.model.exception;
+
+public class AuthorDoesNotExistException extends ObjectDoesNotExistException {
+    public AuthorDoesNotExistException(long id) {
+        super("Nie ma autora o takim ID: " + id);
```

Comment:
nie piszemy kodu po polsku

Reply:
to jest message do wyświetlenia użytkownikowi